First time i built this i got this error:

<details>

```
tandoor-recipes> ________ ERROR at setup of test_search_count[found_recipe1-timescooked] ________
tandoor-recipes> 
tandoor-recipes> self = <QuerySet [<User: jaime97>, <User: djohnson>]>
tandoor-recipes> defaults = {'email': 'megansolomon@example.net', 'first_name': 'Jackson', 'last_name': 'Phillips'}
tandoor-recipes> kwargs = {'space': <Space: board>, 'username': 'djohnson'}
tandoor-recipes> params = {'email': 'megansolomon@example.net', 'first_name': 'Jackson', 'last_name': 'Phillips', 'space': <Space: board>, ...}
tandoor-recipes> 
tandoor-recipes>     def get_or_create(self, defaults=None, **kwargs):
tandoor-recipes>         """
tandoor-recipes>         Look up an object with the given kwargs, creating one if necessary.
tandoor-recipes>         Return a tuple of (object, created), where created is a boolean
tandoor-recipes>         specifying whether an object was created.
tandoor-recipes>         """
tandoor-recipes>         # The get() needs to be targeted at the write database in order
tandoor-recipes>         # to avoid potential transaction consistency problems.
tandoor-recipes>         self._for_write = True
tandoor-recipes>         try:
tandoor-recipes> >           return self.get(**kwargs), False
tandoor-recipes> 
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/query.py:916: 
tandoor-recipes> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tandoor-recipes> 
tandoor-recipes> self = <QuerySet [<User: jaime97>, <User: djohnson>]>, args = ()
tandoor-recipes> kwargs = {'space': <Space: board>, 'username': 'djohnson'}
tandoor-recipes> clone = <QuerySet []>, limit = 21, num = 0
tandoor-recipes> 
tandoor-recipes>     def get(self, *args, **kwargs):
tandoor-recipes>         """
tandoor-recipes>         Perform the query and return a single object matching the given
tandoor-recipes>         keyword arguments.
tandoor-recipes>         """
tandoor-recipes>         if self.query.combinator and (args or kwargs):
tandoor-recipes>             raise NotSupportedError(
tandoor-recipes>                 "Calling QuerySet.get(...) with filters after %s() is not "
tandoor-recipes>                 "supported." % self.query.combinator
tandoor-recipes>             )
tandoor-recipes>         clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
tandoor-recipes>         if self.query.can_filter() and not self.query.distinct_fields:
tandoor-recipes>             clone = clone.order_by()
tandoor-recipes>         limit = None
tandoor-recipes>         if (
tandoor-recipes>             not clone.query.select_for_update
tandoor-recipes>             or connections[clone.db].features.supports_select_for_update_with_limit
tandoor-recipes>         ):
tandoor-recipes>             limit = MAX_GET_RESULTS
tandoor-recipes>             clone.query.set_limits(high=limit)
tandoor-recipes>         num = len(clone)
tandoor-recipes>         if num == 1:
tandoor-recipes>             return clone._result_cache[0]
tandoor-recipes>         if not num:
tandoor-recipes> >           raise self.model.DoesNotExist(
tandoor-recipes>                 "%s matching query does not exist." % self.model._meta.object_name
tandoor-recipes>             )
tandoor-recipes> E           django.contrib.auth.models.User.DoesNotExist: User matching query does not exist.
tandoor-recipes> 
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/query.py:637: DoesNotExist
tandoor-recipes> 
tandoor-recipes> During handling of the above exception, another exception occurred:
tandoor-recipes> 
tandoor-recipes> self = <django.db.backends.utils.CursorWrapper object at 0x7fffef637c90>
tandoor-recipes> sql = 'INSERT INTO "auth_user" ("password", "last_login", "is_superuser", "username", "first_name", "last_name", "email", "is_staff", "is_active", "date_joined") VALUES (%s,'
tandoor-recipes> params = ('', None, False, 'djohnson', 'Jackson', 'Phillips', ...)
tandoor-recipes> ignored_wrapper_args = (False, {'connection': <DatabaseWrapper vendor='sqlite' alias='default'>, 'cursor': <django.db.backends.utils.CursorWrapper object at 0x7fffef637c90>})
tandoor-recipes> 
tandoor-recipes>     def _execute(self, sql, params, *ignored_wrapper_args):
tandoor-recipes>         self.db.validate_no_broken_transaction()
tandoor-recipes>         with self.db.wrap_database_errors:
tandoor-recipes>             if params is None:
tandoor-recipes>                 # params default might be backend specific.
tandoor-recipes>                 return self.cursor.execute(sql)
tandoor-recipes>             else:
tandoor-recipes> >               return self.cursor.execute(sql, params)
tandoor-recipes> 
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/utils.py:89: 
tandoor-recipes> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tandoor-recipes> 
tandoor-recipes> self = <django.db.backends.sqlite3.base.SQLiteCursorWrapper object at 0x7fffee831ac0>
tandoor-recipes> query = 'INSERT INTO "auth_user" ("password", "last_login", "is_superuser", "username", "first_name", "last_name", "email", "is_staff", "is_active", "date_joined") VALUES (?'
tandoor-recipes> params = ('', None, False, 'djohnson', 'Jackson', 'Phillips', ...)
tandoor-recipes> 
tandoor-recipes>     def execute(self, query, params=None):
tandoor-recipes>         if params is None:
tandoor-recipes>             return super().execute(query)
tandoor-recipes>         # Extract names if params is a mapping, i.e. "pyformat" style is used.
tandoor-recipes>         param_names = list(params) if isinstance(params, Mapping) else None
tandoor-recipes>         query = self.convert_query(query, param_names=param_names)
tandoor-recipes> >       return super().execute(query, params)
tandoor-recipes> E       sqlite3.IntegrityError: UNIQUE constraint failed: auth_user.username
tandoor-recipes> 
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/sqlite3/base.py:328: IntegrityError
tandoor-recipes> 
tandoor-recipes> The above exception was the direct cause of the following exception:
tandoor-recipes> 
tandoor-recipes> request = <SubRequest 'found_recipe' for <Function test_search_count[found_recipe1-timescooked]>>
tandoor-recipes> space_1 = <Space: board>, accent = 'àbçđêf ğĦìĵķĽmñ öPqŕşŧ úvŵxyž'
tandoor-recipes> unaccent = 'abcdef ghijklmn opqrst uvwxyz'
tandoor-recipes> u1_s1 = <django.test.client.Client object at 0x7fffedff3e10>
tandoor-recipes> u2_s1 = <django.test.client.Client object at 0x7ffff015a850>
tandoor-recipes> 
tandoor-recipes>     @pytest.fixture
tandoor-recipes>     def found_recipe(request, space_1, accent, unaccent, u1_s1, u2_s1):
tandoor-recipes>         user1 = auth.get_user(u1_s1)
tandoor-recipes>         user2 = auth.get_user(u2_s1)
tandoor-recipes>         days_3 = timezone.now() - timedelta(days=3)
tandoor-recipes>         days_15 = timezone.now() - timedelta(days=15)
tandoor-recipes>         days_30 = timezone.now() - timedelta(days=30)
tandoor-recipes>         if request.param.get('createdon', None):
tandoor-recipes>             recipe1 = RecipeFactory.create(space=space_1, created_at=days_3)
tandoor-recipes>             recipe2 = RecipeFactory.create(space=space_1, created_at=days_30)
tandoor-recipes>             recipe3 = RecipeFactory.create(space=space_1, created_at=days_15)
tandoor-recipes>     
tandoor-recipes>         else:
tandoor-recipes> >           recipe1 = RecipeFactory.create(space=space_1)
tandoor-recipes> 
tandoor-recipes> cookbook/tests/other/test_recipe_full_text_search.py:113: 
tandoor-recipes> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/base.py:528: in create
tandoor-recipes>     return cls._generate(enums.CREATE_STRATEGY, kwargs)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/django.py:121: in _generate
tandoor-recipes>     return super()._generate(strategy, params)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/base.py:465: in _generate
tandoor-recipes>     return step.build()
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/builder.py:270: in build
tandoor-recipes>     step.resolve(pre)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/builder.py:211: in resolve
tandoor-recipes>     self.attributes[field_name] = getattr(self.stub, field_name)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/builder.py:356: in __getattr__
tandoor-recipes>     value = value.evaluate_pre(
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/declarations.py:67: in evaluate_pre
tandoor-recipes>     return self.evaluate(instance, step, context)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/declarations.py:457: in evaluate
tandoor-recipes>     return step.recurse(subfactory, extra, force_sequence=force_sequence)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/builder.py:228: in recurse
tandoor-recipes>     return builder.build(parent_step=self, force_sequence=force_sequence)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/builder.py:274: in build
tandoor-recipes>     instance = self.factory_meta.instantiate(
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/base.py:317: in instantiate
tandoor-recipes>     return self.factory._create(model, *args, **kwargs)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/django.py:171: in _create
tandoor-recipes>     return cls._get_or_create(model_class, *args, **kwargs)
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/django.py:161: in _get_or_create
tandoor-recipes>     raise e
tandoor-recipes> /nix/store/iymwc0fsd3wps1n2l8a51i4zw91vj870-python3.11-factory-boy-3.3.0/lib/python3.11/site-packages/factory/django.py:144: in _get_or_create
tandoor-recipes>     instance, _created = manager.get_or_create(*args, **key_fields)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/manager.py:87: in manager_method
tandoor-recipes>     return getattr(self.get_queryset(), name)(*args, **kwargs)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/query.py:923: in get_or_create
tandoor-recipes>     return self.create(**params), True
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/query.py:658: in create
tandoor-recipes>     obj.save(force_insert=True, using=self.db)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/contrib/auth/base_user.py:76: in save
tandoor-recipes>     super().save(*args, **kwargs)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/base.py:814: in save
tandoor-recipes>     self.save_base(
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/base.py:877: in save_base
tandoor-recipes>     updated = self._save_table(
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/base.py:1020: in _save_table
tandoor-recipes>     results = self._do_insert(
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/base.py:1061: in _do_insert
tandoor-recipes>     return manager._insert(
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/manager.py:87: in manager_method
tandoor-recipes>     return getattr(self.get_queryset(), name)(*args, **kwargs)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/query.py:1805: in _insert
tandoor-recipes>     return query.get_compiler(using=using).execute_sql(returning_fields)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/models/sql/compiler.py:1822: in execute_sql
tandoor-recipes>     cursor.execute(sql, params)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/utils.py:67: in execute
tandoor-recipes>     return self._execute_with_wrappers(
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/utils.py:80: in _execute_with_wrappers
tandoor-recipes>     return executor(sql, params, many, context)
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/utils.py:84: in _execute
tandoor-recipes>     with self.db.wrap_database_errors:
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/utils.py:91: in __exit__
tandoor-recipes>     raise dj_exc_value.with_traceback(traceback) from exc_value
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/utils.py:89: in _execute
tandoor-recipes>     return self.cursor.execute(sql, params)
tandoor-recipes> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tandoor-recipes> 
tandoor-recipes> self = <django.db.backends.sqlite3.base.SQLiteCursorWrapper object at 0x7fffee831ac0>
tandoor-recipes> query = 'INSERT INTO "auth_user" ("password", "last_login", "is_superuser", "username", "first_name", "last_name", "email", "is_staff", "is_active", "date_joined") VALUES (?'
tandoor-recipes> params = ('', None, False, 'djohnson', 'Jackson', 'Phillips', ...)
tandoor-recipes> 
tandoor-recipes>     def execute(self, query, params=None):
tandoor-recipes>         if params is None:
tandoor-recipes>             return super().execute(query)
tandoor-recipes>         # Extract names if params is a mapping, i.e. "pyformat" style is used.
tandoor-recipes>         param_names = list(params) if isinstance(params, Mapping) else None
tandoor-recipes>         query = self.convert_query(query, param_names=param_names)
tandoor-recipes> >       return super().execute(query, params)
tandoor-recipes> E       django.db.utils.IntegrityError: UNIQUE constraint failed: auth_user.username
tandoor-recipes> 
tandoor-recipes> /nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/db/backends/sqlite3/base.py:328: IntegrityError
```
</details>

Flaky builds are annoying to manually restart on hydra. Please disable `test_search_count` with a comment indicating it is flaky.

---

I seem to get the following error when i try to run it:

```shell
$ result/bin/tandoor-recipes
Traceback (most recent call last):
  File "/nix/store/956cr1xszn3vbvq5bphpbgg6cmfrg81l-python3.11-tandoor-recipes-1.5.10/lib/tandoor-recipes/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/vinfkk30jy2560i66q9pk6nsns9y7jhk-python3.11-Django-4.2.7/lib/python3.11/site-packages/django/apps/config.py", line 193, in create
    import_module(entry)
  File "/nix/store/qp5zys77biz7imbk6yy85q5pdv7qk84j-python3-3.11.6/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'debug_toolbar'
```

--- a/nixos/modules/services/misc/tandoor-recipes.nix
+++ b/nixos/modules/services/misc/tandoor-recipes.nix
@@ -12,7 +12,7 @@ let
     DEBUG_TOOLBAR = "0";
     MEDIA_ROOT = "/var/lib/tandoor-recipes";
   } // optionalAttrs (config.time.timeZone != null) {
-    TIMEZONE = config.time.timeZone;
+    TZ = config.time.timeZone;
   } // (
     lib.mapAttrs (_: toString) cfg.extraConfig
   );

--- a/pkgs/applications/misc/tandoor-recipes/common.nix
+++ b/pkgs/applications/misc/tandoor-recipes/common.nix
@@ -1,15 +1,15 @@
 { lib, fetchFromGitHub }:
 rec {
-  version = "1.5.6";
+  version = "1.5.10";
 
   src = fetchFromGitHub {
     owner = "TandoorRecipes";
     repo = "recipes";
     rev = version;
-    hash = "sha256-3sitrTaIRKmjx+5vWOQXE0/Gu0jJ8VCpOq2cZZVLrbk=";
+    hash = "sha256-CkqNPG57e76TT/vF9lscS6m2FbXOvOfqiT/9aM2Il9E=";
   };
 
-  yarnHash = "sha256-mZ8beCF+3mnpgKED0fP96RBbGbKNNXqEJkGSjgrEGBc=";
+  yarnHash = "sha256-atl2XrY9LmWh2USp6K2W50/khEsnY6OqKBUS26Ln9ZM=";
 
   meta = with lib; {
     homepage = "https://tandoor.dev/";

--- a/pkgs/applications/misc/tandoor-recipes/default.nix
+++ b/pkgs/applications/misc/tandoor-recipes/default.nix
@@ -23,8 +23,8 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     ./media-root.patch
     # https://github.com/TandoorRecipes/recipes/pull/2706
     (fetchpatch {
-      url = "https://github.com/TandoorRecipes/recipes/commit/8f66f5c3ca61751a80cc133ff4c59019d6fca406.patch";
-      hash = "sha256-oF5YlPg1LEdLvKpxiSqjTmYPbrGquPlRIz6A05031gs=";
+      url = "https://github.com/TandoorRecipes/recipes/commit/702c1d67d3b2d13cf471bf9daa1d2ef0f1837dec.patch";
+      hash = "sha256-6vmtYs6b0d38Ojxxc2I7oxqpkIlyRVlhzURBOTO2VlQ=";
     })
   ];
 
@@ -42,6 +42,7 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     django-cleanup
     django-cors-headers
     django-crispy-forms
+    django-crispy-bootstrap4
     django-hcaptcha
     django-js-reverse
     django-oauth-toolkit
@@ -122,6 +123,11 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     pytestCheckHook
     pytest-django
     pytest-factoryboy
+  ];
+
+  # flaky
+  disabledTests = [
+    "test_search_count"
   ];
 
   passthru = {

